src/Stratis.SmartContracts.CLR/CallDataSerializer.cs

```diff
@@ -182,5 +197,46 @@ protected object[] DeserializeMethodParameters(byte[] methodParametersRaw)
 
             return methodParameters;
         }
+
+        protected byte[] SerializeSignatures(string[] signatures)
+        {
+            byte[][] signaturesRaw = new byte[signatures.Length][];
+            int totalBytes = 0;
+            for (int i = 0; i < signatures.Length; i++)
+            {
+                signaturesRaw[i] = Convert.FromBase64String(signatures[i]);
+                totalBytes += signaturesRaw[i].Length + 1;
```

Comment:
Why `+1`

Reply:
Each signature is prefixed by the length thereof.